test-biblatex.tex

```diff
@@ -16,6 +16,7 @@
 
 \RequirePackage[utf8]{inputenc}
 \RequirePackage[T1,T2A]{fontenc}
+\RequirePackage{textcomp}
```

Comment:
Package inputenc Error: Unicode char \u8:× not set up for use with LaTeX. That's what I had without textcomp